scripts/read_spectrum.R

```diff
@@ -11,9 +11,18 @@ prepare_hc_leapfrog_projp <- function(pjnz, params, pop_1){
   v$paed_cd4_dist <- dp_read_paed_cd4_dist(pjnz) / 100
 
   prog = dp_read_paed_cd4_prog(pjnz)
-  paed_cd4_prog <- array(c(prog[1,1:6],0), dim = 7, dimnames = list(cd4 = c('30plus', '26-30', '21-25', '16-20', '11-15', '5-10', '<5')))
+  paed_cd4_prog <- array(0, dim = c(7,2,2), dimnames = list(cd4 = c('30plus', '26-30', '21-25', '16-20', '11-15', '5-10', '<5'),
+                                                            age = c('0-2', '3-4'),
+                                                            sex = c('male', 'female')))
+  paed_cd4_prog[,1,1] <- c(prog[1,1:6], 0)
+  paed_cd4_prog[,1,2] <- c(prog[2,1:6], 0)
 
-  adol_cd4_prog <- array(c(prog[1,14:18],0), dim = 6, dimnames = list(cd4 = c('>1000', '750-999', '500-749', '350-499', '200-349', 'lte200')))
```

Comment:
calling 5-14 year olds 'adolescents' is a bit of a stretch. But I guess it's a simple clear distinction.